pkg/ctl/utils/update_cluster_endpoint_access.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+)
+
+func updateEndpointAccessCmd(cmd *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	cmd.ClusterConfig = cfg
+
+	cmd.SetDescription("update-cluster-api-access", "Update cluster API endpoint configuration", "")
+
+	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+		cmdutils.AddNameFlag(fs, cfg.Metadata)
+		cmdutils.AddRegionFlag(fs, cmd.ProviderConfig)
+		cmdutils.AddConfigFileFlag(fs, &cmd.ClusterConfigFile)
+		cmdutils.AddApproveFlag(fs, cmd)
+		cmdutils.AddTimeoutFlag(fs, &cmd.ProviderConfig.WaitTimeout)
+	})
+
+	var private, public bool
+	cmd.FlagSetGroup.InFlagSet("Allow/Disallow Cluster API access flags",
```

Comment:
- word "flags" will be appended by the library
- 'Cluster API' means something else, let's avoid using it here

```suggestion
	cmd.FlagSetGroup.InFlagSet("Update private/public Kubernetes API endpoint configuration",
```

Reply:
Seems like another case were "endpoint configuration" might be "endpoint access configuration"?

Reply:
Up to you, as long as it's consistent :)